.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - "7.5.0"
+env:
+  - NPM_CONFIG_LOGLEVEL=warn
+install:
+  - npm install
+  - npm run bootstrap
```

Comment:
Do we need this? `bootstrap` is run as postinstall script, meaning `npm install` should be sufficient. However, for some reason, Drone doesn't do it. Is that the same for Travis?